guides/theming.md

```diff
@@ -134,7 +134,8 @@ With this, any element inside of a parent with the `unicorn-dark-theme` class wi
 dark theme.
 
 ### Theming your own components
-For more details about theming your own components, see [theming-your-components.md](https://github.com/angular/material2/blob/master/guides/theming-your-components.md)
+For more details about theming your own components, 
+see [theming-your-components.md](./theming-your-components.md)
```

Comment:
There's a bit of code in the `gulp docs` task that makes the links point to the right place on the docs site.